tests/test_tictactoe.py

```diff
@@ -10,6 +10,23 @@ def tt():
     return tictactoe.TicTacToeEnv()
 
 
+def test_policy_page_lines():
+    """
+    Just a calling a policy to make sure that it replay from a couple of different boards.
+    """
+    game = tictactoe.TicTacToeEnv()
```

Comment:
suggestion: you can add `tt` to the arguments and it'll give you a tictactoenv (using pytest fixtures) - it's what I've done elsehwere

Reply:
Yes, I saw the fixture, but for some reason thought that ways in which you intended to vary the tt fixture might not be required for this test... I'll reassess, perhaps I'm imagining things.

Reply:
No real need, it's just a little syntactic sugar.